### PR TITLE
Timeout issue

### PR DESCRIPTION
**Pull request by malandrew**
This pull request contains two changes:

(1) `once()` changed to `on()` in index.js so you can run one test server and visit from multiple browsers (in the future I reckon you could have the browser return user agent info to decorate tests from multiple sources. However, once this point is reached, it's probably time for a refactor/rewrite with full tests)

(2) global timeout option so that you can fail the tests if they take too long. To keep in line with the prior behavior, the timeout is infinity by default, but most people might want to shorten this to 10 or 30 seconds. 

I will be producing a separate library for adding timers to individual tape tests since that is beyond the scope of run-browser.

### CODE REVIEW

**Comment by malandrew** [comment]
In addition to the changes above, I removed the debug option and moved the TAP output to be printed out in streaming mode so you know what is going on with long running tests.

**Comment by Raynos** [comment]
This debug feature sounds useful :P

**Comment by Raynos** [comment]
The new timeout changes look good.

**Comment by malandrew** [comment]
Split up into three pull requests for each feature:

https://github.com/ForbesLindesay/run-browser/pull/8
https://github.com/ForbesLindesay/run-browser/pull/9
https://github.com/ForbesLindesay/run-browser/pull/10

**Comment by malandrew** [comment]
FWIW, the debug feature sounds more useful than it is.... Previously I would accumulate the console.log output in the browser and the send it to the server to be written to the terminal. The debug feature just printed the exact same output to the terminal as it was generated instead of when the tests finish. These changes make the "debug" option the default behavior and eliminates the accumulation then printing of output. 

Basically, while the name "debug" implies that it was something more verbose and useful, it actually is not.
